api/src/main/resources/liquibase.xml

```diff
@@ -149,4 +149,52 @@
             UPDATE obs SET value_coded = 160546 WHERE concept_id = 165839 AND value_coded = 160548 ;
         </sql>
     </changeSet>
+    <changeSet id="add_risk_stratification_adult_encounter_type_202102220223" author="arinze">
```

Comment:
add a preconditon to this so it won't fail for database that already contains the encounter type. this is how you add a preconditon 

 <changeSet id="include_AZT-3TC-LPV/r_202101241029" author="wande" runOnChange="true">
        <preConditions onFail="MARK_RAN">
            <sqlCheck expectedResult="0">
                SELECT COUNT(*) FROM concept_answer WHERE concept_id='164507' AND  answer_concept='162561';
            </sqlCheck>
        </preConditions>
        <insert tableName="concept_answer">
            <column name="concept_id" valueNumeric="164507"/>
            <column name="answer_concept" valueNumeric="162561"/>
            <column name="creator" valueNumeric="1"/>
            <column name="date_created" valueDate="now()"/>
            <column name="uuid" value="133caea0-4dca-4767-98e2-ab2ff2022c48"/>
            <column name="sort_weight" valueNumeric="16"/>
        </insert>
    </changeSet>

in this case you'll be checking for the uuid